### ISSUE TITLE
[ FEAT ] : Adding Achievements section in the template

### ISSUE THREAD

**Issue by 1-SubhamSingh** [koalalu007/resume-maker#49]
Adding an achievements section can elevate the content of the resume and also there is no section for mentioning some achievements.

I'll complete this task , can you  please assign me this task under the hacktoberfest tag :)

**Comment by koalalu007** [comment]
are you working?

**Comment by 1-SubhamSingh** [comment]
> are you working?

Yes , will update with the complete task tomorrow or maybe by tonight.

**Comment by 1-SubhamSingh** [comment]
@koalalu007  I have fixed it could you please check and merge the PR
